tools/crowdin/actions/unhide-specific-string/index.js

```diff
@@ -0,0 +1,34 @@
+require('dotenv').config({ path: `${__dirname}/../../.env` });
+const core = require('@actions/core');
+const { getFiles } = require('../../utils/files');
+const { getStrings, changeHiddenStatus } = require('../../utils/strings');
+// eslint-disable-next-line import/no-unresolved
+
+const filename = core.getInput('filename');
+const stringContent = core.getInput('string-content');
+
+const hideString = async (projectId, fileName, string) => {
+  const fileResponse = await getFiles(projectId);
+  const targetFile = fileResponse.find(el => el.path.endsWith(filename));
```

Comment:
```suggestion
  const targetFile = fileResponse.find(el => el.path.endsWith(filename));
```
How about `path.baseName(el.path) === filename`?  I doubt the above code will ever be an issue, but I think `baseName` is a bit more robust.